django/core/cache/backends/db.py

```diff
@@ -202,15 +202,29 @@ def _base_set(self, mode, key, value, timeout=DEFAULT_TIMEOUT):
                 return True
 
     def delete(self, key, version=None):
-        key = self.make_key(key, version=version)
-        self.validate_key(key)
```

Comment:
Where is this called in `delete_many()`? Can you explain why it is not needed?

Reply:
It is needed

Reply:
I figured - I was making the point that it was missing and that the removal ought to be explained if it was intentional... :stuck_out_tongue: